### ISSUE TITLE
Custom sounds not working due to typo

### ISSUE THREAD

**Issue by WhoTHCares** [WeakAuras/WeakAuras2#181]
What steps will reproduce the problem?  Adding sounds with a full path do not work.

What is the expected output? What do you see instead?  Custom sounds cannot be played.

Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?  This is an easy fix and not affected by any other addons.

What version of WeakAuras are you using?   v2.3.9

Was it working in a previous version? If yes, which was the last good one?  Unknown

Do you have an error log of what happened? If you don't see any errors, make sure that error reporting is enabled (`/run scriptErrors 1`) or install [BugSack](https://mods.curse.com/addons/wow/BugSack). 

Please provide any additional information below.  
Line 2175 of "WeakAuras.lua" reads as-follows:
   if(actions.sound == " custom") then
It should read:
   if(actions.sound == "Custom") then

Once the entry above is changed and the file is saved, restart WoW and custom sounds will play.
I loaded my sounds in "interface\addons\wasounds" and referenced the Sound File Path as "interface\addons\wasounds\filename.ogg"

If you do not know which aura is causing issues for you, please attach a ZIP archive of your WeakAuras SavedVariables file. It's the `WeakAuras.lua` file in `World of Warcraft\WTF\Account\YOUR_ACCOUNT\SavedVariables\`

In case you do know which aura is problematic, please export the string and paste it below.

```  HERE ```

**Comment by InfusOnWoW** [comment]
Your analysis is incorrect. That line is correct. See WeakAuras.sound_types in Types.lua, which contains " custom".

**Comment by WhoTHCares** [comment]
So why do custom sounds NOT play until I changed it as I described?  

I modified three files in total, WeakAuras/Types.lua (changed line 1242), WeakAurasOptions\WeakAurasOptions.lua (changed lines 2271 and 2453) and WeakAuras/WeakAuras.lua (changed line 2175) all to match from " custom" to "Custom" without a space as I described in my original post.

Maybe case does not matter, but the space is clearly throwing something off.  I now have custom sounds that work where I did not have working custom sounds before.  The ogg files I have loaded now play on pressing Okay as well as when my action is initiated.
